README.md

```diff
@@ -420,6 +421,93 @@ status:
 - `$.status.conditions[?(@.type == "Failed")].status == "True"` means that a failure condition has been fulfilled and the resource is considered failed.
 - Since `error_msg_path` is specified, kubernetes-deploy will log the contents of `$.status.conditions[?(@.type == "Failed")].message`, which in this case is: `resource is failed`.
 
+### Deploy walkthrough
+
+Let's walk through what happens when you run the `deploy` task with [this directory of templates](https://github.com/Shopify/kubernetes-deploy/tree/master/test/fixtures/hello-cloud). You can see this for yourself by running the following command:
+
+```bash
+krane deploy -f test/fixtures/hello-cloud --render-erb my-namespace my-k8s-cluster
```

Comment:
Is `krane deploy my-namespace my-k8s-cluster -f test/fixtures/hello-cloud --render-erb ` easier to understand?